packages/core/src/modules/dids/__tests__/peer-did.test.ts

```diff
@@ -0,0 +1,157 @@
+import type { IndyLedgerService } from '../../ledger'
+
+import { getAgentConfig } from '../../../../tests/helpers'
```

Comment:
@jakubkoci I wrote these tests to approximately work out what we'll be needing for DIDExchange. The first test is when we create the did document. The second step is when we are receiving the did document